src/main/resources/application.properties

```
@@ -24,3 +24,6 @@ material1.mops.storage.minio.accessKey=${MINIO_ACCESS_KEY:dev_minio_access_key}
 material1.mops.storage.minio.secretKey=${MINIO_SECRET_KEY:dev_minio_secret_key}
 
 material1.mops.gruppenfindung.url=${MATERIAL1_GRUPPENFINDUNG_URL:https://mops.hhu.de/gruppe1}
+
+material1.mops.configuration.admin=${MATERIAL1_ADMIN_ROLE:admin}
+material1.mops.configuration.max-groups=${MATERIAL_MAX_FOLDER_PER_GROUP:200L}
```

I think the L must not be here
```suggestion
material1.mops.configuration.max-groups=${MATERIAL_MAX_FOLDER_PER_GROUP:200}
```